citation_helper_service/tests/test_webservices.py

```diff
@@ -38,12 +39,12 @@ def test_ResourcesRoute(self):
 
         for expected_field, _type in {'scopes': list, 'methods': list,
                                       'description': basestring,
-                                      'rate_limit': list}.iteritems():
+                                      'rate_limit': list}.items():
             # Assert each resource is described has the expected_field
-            [self.assertIn(expected_field, v) for v in r.json.values()]
+            [self.assertIn(expected_field, v) for v in list(r.json.values())]
```

Comment:
I think this `list(...)` (automatically added by the porting tools) might not be needed here.